sdk/src/main/java/com/horizen/cryptolibprovider/ThresholdSignatureCircuitImplZendoo.java

```diff
@@ -144,7 +137,7 @@ public Pair<byte[], Long> createProof(List<WithdrawalRequestBox> bt,
     }
 
     @Override
-    public Boolean verifyProof(List<WithdrawalRequestBox> bt,
+    public Boolean verifyProof(List<BackwardTransfer> backwardTransfers,
```

Comment:
This note is for the TODO comment above (line 129):
According to Daniele that would be a useful optimization, it might be turned into a task.